src/EnergyPlus/StandardRatings.cc

```diff
@@ -990,7 +991,7 @@ namespace StandardRatings {
                                                    EIRFFlowCurveIndex(1),
                                                    PLFFPLRCurveIndex(1));
 
-                // Calculated Net Cooling Capacity, SEER, EER, and IEER of single speed DX cooling coils
+                // Calculated Net Cooling Capacity, SEER, SEER Default, EER, and IEER of single speed DX cooling coils
                 SingelSpeedDXCoolingCoilStandardRatings(DXCoilName,
```

Comment:
Would be nice to fix the spelling of this function name, since you're touching this code anyway.

Reply:
Done!